packages/patapata_core/test/standard_app_widget_tab_test.dart

```diff
@@ -587,8 +587,10 @@ class _TestTitlePage extends StandardPage<void> {
       appBar: AppBar(
         title: const Text("Test Title Page"),
         automaticallyImplyLeading: false,
-        leading: const StandardPageBackButton(
-          key: ValueKey(_kTestStandardPageBackButton),
+        // Purposely not using `const` to improve code coverage.
+        // ignore: prefer_const_constructors
```

Comment:
```suggestion
        // Purposely not using `const` because the current version of flutter is flagging const constructors as not being executed in coverage tests.
        // ignore: prefer_const_constructors
```